scripts/chrome/driver-install.sh

```diff
@@ -10,7 +10,7 @@ if [ ! -f $HOME/$CHROMEDRIVER_ZIP ]; then
     wget --quiet -O $HOME/$CHROMEDRIVER_ZIP $CHROMEDRIVER_URL
 fi
 
-if ! $HOME/bin/chromedriver --version | grep -q "^ChromeDriver ${CHROMEDRIVER_VERSION//\./\\.}"; then
+if [ ! -f $HOME/bin/chromedriver ] || ! $HOME/bin/chromedriver --version | grep -q "^ChromeDriver ${CHROMEDRIVER_VERSION//\./\\.}"; then
```

Comment:
Get rid of a `file not found` warning just to make the provision output cleaner (less red).

Reply:
+1